shipyard-controller/handler/sequencehooks/sequencestate.go

```diff
@@ -49,6 +49,13 @@ func (smv *SequenceStateMaterializedView) OnSequenceTriggered(event models.Event
 		State:          models.SequenceTriggeredState,
 		Stages:         []models.SequenceStateStage{},
 	}
+
+	//if the next event in sequence is an action we get the problem title form it
+	getActionTriggeredData := &keptnv2.GetActionTriggeredEventData{}
```

Comment:
please add a unit test for that case in `shipyard-controller/handler/sequencehooks/sequencestate_test.go` as well :)

Reply:
sure thing I'll make one!